src/pollution_model/run_pollution_model.py

```diff
@@ -0,0 +1,454 @@
+"""
+This script takes in appropriate datasets as input and runs the MEDUSA2.0 model to calculate
+TSS (total suspended solids), TCu (total copper), DCu  (dissolved copper), TZn (total zinc), and DZn (dissolved zinc).
+"""
+
+import logging
+import math
+import os
+import pathlib
+import platform
+import subprocess
+from datetime import datetime
+from typing import Tuple, Union, Optional, TextIO
+from enum import Enum
+
+import geopandas as gpd
+import numpy as np
+import sqlalchemy
+import xarray as xr
+from newzealidar.utils import get_dem_by_geometry
+from sqlalchemy import insert
+from sqlalchemy.engine import Engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.sql import text
+
+from src import config
+from src.digitaltwin import setup_environment
+from src.digitaltwin.tables import BGFloodModelOutput, create_table, check_table_exists
+from src.digitaltwin.utils import LogLevel, setup_logging, get_catchment_area
+from src.flood_model.flooded_buildings import find_flooded_buildings
+from src.flood_model.flooded_buildings import store_flooded_buildings_in_database
+from src.flood_model.serve_model import add_model_output_to_geoserver
+
+log = logging.getLogger(__name__)
+
+Base = declarative_base()
+
+# Testing to commit
+
+class SurfaceType(Enum):
+    ConcreteRoof = 1
+    CopperRoof = 2
+    GalvanisedRoof = 3
+    AsphaltRoad = 4
+    #CarPark = 5 - CarParks are classified the same as roads

```

Comment:
Would recommend considering using StrEnum, like casey used in `tide_enum.py`
In addition to this, please use `UPPER_CASE_FOR_ENUMS`